Reading List/Reading List/Model Controller/BookController.swift

```diff
@@ -0,0 +1,127 @@
+//
+//  BookController.swift
+//  Reading List
+//
+//  Created by Norlan Tibanear on 6/6/20.
+//  Copyright © 2020 Lambda School. All rights reserved.
+//
+
+import Foundation
+
+class BookController {
```

Comment:
Just want to give you a virtual pat on the back for getting this controller working. I know it's not easy and some of the code still hasn't sunk in 100%, but great job getting it functioning!